src/L8/game/checkers/checkers_game.py

```diff
@@ -11,15 +11,27 @@
 class CheckersGame(Game, ABC):
 
     def __init__(self, board: Board, players: list):
-        super().__init__(board, players)
+        super().__init__(board, players, True)
 
     def is_valid_move(self, move: dict, player: Player) -> bool:
-        r1, c1, r2, c2, is_jump, player_color = move[MOVE]
-        move = CheckersMove(r1, c1, r2, c2, is_jump)
+        r1, c1, r2, c2 = move[MOVE]
+        if player.game_token is "kw":
+            player_color = "w"
+        elif player.game_token is "kb":
+            player_color = "b"
+        else:
+            player_color = player.game_token
```

Comment:
Here you are storing player_color with a GameToken object, but on lines 19 and 21 you are storing a str in the same variable.

I would recommend that on lines 19 and 21 you get the actual singleton GameToken object. I believe there is a util method that can fetch you a GameToken instance given a str representation of said token, if there is not a method, we could add it.

By not keeping the typing consistent, you might run into issues if you are using if's to compare values, after all, a str object will not evaluate to True against a GameToken object (unless you transform either to the other value).